### PR TITLE
WIP Improve load page speed

### PR DESCRIPTION
**Pull request by suecarmol**
## Description
Started adding the organization context statistics programmatically, instead of querying the database multiple times

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This is to improve Wikilink's performance since some pages do not load.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T240673](https://phabricator.wikimedia.org/T240673)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by Samwalton9** [comment]
Would it make sense to break out the formatting changes to a different PR? It makes this one hard to follow for the 'real' changes!

**Comment by jsnshrmn** [comment]
Agree with @Samwalton9. I can see that you wrote some new helpers and reverted some mysql behavior with a settings change, but a substantial portion of the diff in this pr is not about the performance issue at all. I'd recommend either dropping all of the formatting changes here, or going back to master and running black to reformat everything and then doing this pr against that.

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
Agree with @Samwalton9. I can see that you wrote some new helpers and reverted some mysql behavior with a settings change, but a substantial portion of the diff in this pr is not about the performance issue at all. I'd recommend either dropping all of the formatting changes here, or going back to master and running black to reformat everything and then doing this pr against that.

**Comment by suecarmol** [comment]
Closing out this PR in favor of creating aggregate tables instead of programatically trying to improve performance
